content/chat/channels.textile

```diff
@@ -0,0 +1,105 @@
+---
+title: Channels and Messages
+meta_description: "Channels bring forward the capability to provide real-time communication between multiple people."
+product: chat
+languages:
+  - javascript
+---
+
+Channels are the core functionality for any chat application or any application with chat features. Channels enable you to separate chat channels and their messages into categories. For example, a single channel for direct messages between two people, or a channel for a specific live streaming event with many chat participants.
```

Comment:
"For example, a **single** channel for direct messages between two people, or a channel for a specific live streaming event with many chat participants."

Remove single

Reply:
Removed: https://github.com/ably/docs/pull/2090/commits/8b19a80343647dcfc21f0fbfe2b0116b9b5274a9